### PR TITLE
Add SpringInterpolator

### PR DESCRIPTION
**Pull request by imkunet**
This is an example of a pull request. This text area supports markdown!

### Try titles
**or bolded text**
~~the possibilities are endless~~

Write about what you add to the project with your pull request. Make sure to be descriptive!
Contributors will be able to add comments and if you allow it edit your pull request too.

Make sure to always follow the Pull Request guidelines of the repository you're contributing too and remain civil!

### CODE REVIEW

**Comment by imkunet** [comment]
- [x] @imkunet  approved these changes

**Comment by imkunet** [comment]
There are no mistakes in the JSON file but even if there were they were fixed in the last commit

File: Repository/0.5.3.1/v0.0.1.json [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+    "Owner": "imkunet",
+    "Description": "Interpolates your pointer based on a spring physics system",
+    "PluginVersion": "0.0.1",
+    "SupportedDriverVersion": "0.5.3.1",
+    "RepositoryUrl": "https://github.com/imkunet/SpringInterpolator",
+    "DownloadUrl": "https://github.com/imkunet/SpringInterpolator/releases/download/0.0.1/SpringInterpolator.zip",
+    "CompressionFormat": "zip",
+    "SHA256": "2a41a53fb584788b3bf219b7a3cddbfe48e883693ffa22b960928b1f3f49aede",
+    "WikiUrl": "https://github.com/imkunet/SpringInterpolator/blob/master/README.md",
+    "LicenseIdentifier": "Unilicense"
```
Comment (InfinityGhost) [inline Repository/0.5.3.1/v0.0.1.json:12]: Typo
```suggestion
    "LicenseIdentifier": "Unlicense"
```

**Review by InfinityGhost — CHANGES_REQUESTED** [review]
`Repository/0.5.3.1/v0.0.1.json` is not the correct path, it should be `Repository/0.5.3.1/imkunet/SpringInterpolator/v0.0.1.json`

Fix the other issue and this can be merged.

File: Repository/0.5.3.1/v0.0.1.json (hunk 1 above)
Comment (AbstractQbit) [reply Repository/0.5.3.1/v0.0.1.json:12]: Actually the path should be `Repository/0.5.3.1/imkunet/SpringInterpolator/SpringInterpolator/v0.0.1.json`

Comment (InfinityGhost) [reply Repository/0.5.3.1/v0.0.1.json:12]: Follow AbstractQbit, I apparently don't even know my own directory schema

**Review by InfinityGhost — APPROVED** [review]

**Pull request merged**